src/data/actions/courseInfo.js

```diff
@@ -66,15 +81,52 @@ export function fetchCourseInfo(id) {
   };
 }
 
-export function createCourse(courseData) {
+export function createCourseRun(course, courseRunData) {
+  return (dispatch) => {
+    dispatch(createNewCourseRun());
+    if (!course.key) {
+      dispatch(courseCreateRunFail('Course Run create failed, please try again or contact support. Error( Course create incomplete )'));
+      return null;
+    }
+
+    const data = {
+      course: course.key,
+      start: courseRunData.start,
+      end: courseRunData.end,
+    };
+    return DiscoveryDataApiService.createCourseRun(data)
+      .then((response) => {
+        const courseRun = response.data;
+        dispatch(courseCreateRunSuccess(courseRun));
+        return dispatch(push(`/courses/${course.uuid}/`));
```

Comment:
Shouldn't this be .../edit/?

Reply:
/edit does not exist in this PR :D need to wait for yours to merge

Reply:
Plan ahead! :)

Reply:
Going to rebase and do this once your PR merges
👍